onnx/backend/test/case/node/hardswish.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+from onnx.backend.sample.ops.abs import abs
+
+
+class HardSwish(Base):
+
+    @staticmethod
+    def export():  # type: () -> None
+        node = onnx.helper.make_node(
+            'HardSwish',
+            inputs=['x'],
+            outputs=['y'],
+        )
+        x = np.random.randn(3, 4, 5).astype(np.float32)
+        alfa = float(1 / 6)
+        beta = 0.5
+        y = x * np.maximum(0, np.minimum(1, alfa * x + beta))
```

Comment:
Lifting these 3 lines out as a separate function "hard_swish" would be helpful to other users as a reference implementation. (Eg., like here: https://github.com/onnx/onnx/blob/03e6c8f2ad599dda614a22ee494e982ffd0bccc7/onnx/backend/test/case/node/logsoftmax.py#L15 )

Reply:
Done